modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor5/plugins/WriteAssistent/components/WriteAssistentActions.tsx

```diff
@@ -0,0 +1,154 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayDropDown from '@clayui/drop-down';
+import {fetch} from 'frontend-js-web';
+import React, {useEffect, useRef, useState} from 'react';
+
+interface ActionItem {
+	disabled?: boolean;
+	name: string;
+	symbolLeft?: string;
+	symbolRight?: string;
+	type:
+		| 'Improve Writing'
+		| 'Fix Spelling Grammar'
+		| 'Translate To'
+		| 'Make Shorter'
+		| 'Make Longer'
+		| 'Generate Based On Title';
+}
+
+export default function WriteAssistentActions({
+	containerRef,
+	content,
+}: {
+	content: string;
+	containerRef: HTMLElement;
+}) {
+	const [active, setActive] = useState(true);
+
+	const actionsGroup = [
+		{
+			children: [
+				{
+					disabled: false,
+					name: Liferay.Language.get('improve-writing'),
+					symbolLeft: 'magic',
+					type: 'Improve Writing',
+				},
+				{
+					disabled: true,
+					name: Liferay.Language.get('fix-spelling-grammar'),
+					symbolLeft: 'check',
+					type: 'Fix Spelling Grammar',
```

Comment:
@igor-franca I'm not sure if is already the value to be passed to the request, but if so it should be: 

Fix Spelling **and** Grammar

Thanks!